### PR TITLE
Add object lager sink

### PR DESCRIPTION
**Pull request by bsparrow435**
Also add a version entry into riak_kv_entropy_manager ets table so we
don't need to serialize get_version requests on the managers msgq.

### CODE REVIEW

File: src/riak_kv_get_core.erl [hunk 1]
```diff
@@ -310,7 +310,7 @@ maybe_log_old_vclock(Results) ->
                         [] ->
                             ok;
                         _ ->
-                            lager:warning("Bucket: ~p Key: ~p should be rewritten to guarantee
+                            object:warning("Bucket: ~p Key: ~p should be rewritten to guarantee
```
Comment (hazen) [inline src/riak_kv_get_core.erl:313]: Dialyzer doesn't seem too happy with this.  Is this change intentional?

Comment (bsparrow435) [reply src/riak_kv_get_core.erl:313]: Yes, lager3 will do a parse transform on object: to route to the correct sink. Purpose of the change in rebar.config: {lager_extra_sinks, [object]}

**Comment by JeetKunDoug** [comment]
+1 05866ee

**Comment by hazen** [comment]
@borshop merge
